alfred-telemetry-platform declares transitive dependency on Alfresco enterprise

Hi,

I'd like to use alfred-telemetry, it looks really interesting, thanks for sharing it! I followed the README, but I'm having troubles locating the AMP files; [the link to Maven Central](https://search.maven.org/search?q=g:eu.xenit.alfred.telemetry%20AND%20a:alfred-telemetry-platform) only contain the POM files.

Thanks in advance!

Thanks for the quick response @kerkhofsd !

I still have issues defining it as a dependency; I managed to isolate the issue: if I run `mvn package` on the pom.xml below, I get an error. Do you also have the same issue?

```
<?xml version="1.0" encoding="UTF-8"?>
<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
    <modelVersion>4.0.0</modelVersion>
    <artifactId>sample</artifactId>
    <groupId>com.acme</groupId>
    <version>0.1-SNAPSHOT</version>
    <dependencies>
        <dependency>
            <groupId>eu.xenit.alfred.telemetry</groupId>
            <artifactId>alfred-telemetry-platform</artifactId>
            <version>0.3.1</version>
            <type>amp</type>
        </dependency>        
    </dependencies>
</project>
```

Error below:
```
[INFO] Scanning for projects...
[INFO] 
[INFO] --------------------------< com.acme:sample >---------------------------
[INFO] Building sample 0.1-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.350 s
[INFO] Finished at: 2020-12-25T05:06:41+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project sample: Could not resolve dependencies for project com.acme:sample:jar:0.1-SNAPSHOT: Failed to collect dependencies at eu.xenit.alfred.telemetry:alfred-telemetry-platform:amp:0.3.1: Failed to read artifact descriptor for eu.xenit.alfred.telemetry:alfred-telemetry-platform:amp:0.3.1: Failure to find org.alfresco:acs-packaging:pom:6.1.0.5 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

Seems that it can't find `org.alfresco:acs-packaging:pom:6.1.0.5`, which in fact is not listed on https://mvnrepository.com/artifact/org.alfresco/acs-community-packaging . 

Would be possible to update https://github.com/xenit-eu/alfred-telemetry/blob/master/alfred-telemetry-platform/build.gradle#L25 to `6.1.1` (or better, `6.1.2-ga`) ?

Thanks!

--- a/alfred-telemetry-platform/build.gradle
+++ b/alfred-telemetry-platform/build.gradle
@@ -26,7 +26,7 @@ ext {
 }
 
 dependencies {
-    implementation enforcedPlatform("org.alfresco:acs-packaging:${alfrescoVersion}")
+    alfrescoProvided enforcedPlatform("org.alfresco:acs-packaging:${alfrescoVersion}")
 
     alfrescoProvided('org.alfresco:alfresco-repository')
 

